engine/src/exec-interface2.cpp

```diff
@@ -2480,8 +2480,18 @@ bool MCInterfaceStringCouldBeStack(MCStringRef p_string)
          MCStringBeginsWithCString(p_string, (const char_t *)"REVO",
                                   kMCCompareExact)))
         return true;
-    
-    // Check if it could be a script-only stack
+
+    //Here we check whether it is very likely that the string is a UTF-8
+    //encoded script-only stack and not a name. Any UTF-8 encoded script-only
+    //stack with BOM will start with 'BOMscript "'. - i.e. have the 3 byte
+    //BOM prefix and then 'script "'.
+    const char_t t_UTF8BOM[] = { 0xEF, 0xBB, 0xBF, 's', 'c', 'r', 'i', 'p',
```

Comment:
I suggest perhaps using k_ prefix rather than t_ here - as it is a constant (I'm not sure we cover const .. type vars in our style guide at the moment). Also, generally lower-case var names are preferred - k_utf8_bom.